node/src/test/java/ch/prevo/open/node/config/NodeConfigurationServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package ch.prevo.open.node.config;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.inject.Inject;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
```

Comment:
Could you try if `+@SpringBootTest(classes = {NodeConfigurationServiceTest.class })` would be faster? because I don't think that it is necessary for this test to start the whole application context.

Reply:
Slightly faster, 
done